assets/js/blocks/product-results-count/style.scss

```diff
@@ -5,5 +5,21 @@
 	.woocommerce-result-count {
 		float: none;
 		font-size: inherit;
+		// reset for margin
+		margin: auto;
 	}
 }
+
+.woocommerce .wc-block-catalog-sorting {
+	.woocommerce-ordering {
+		margin: auto;
+	}
+}
```

Comment:
It would be great to move the Catalog Sorting styles to the respective style file: https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/blocks/catalog-sorting/style.scss

Reply:
Oh yea definitely, my bad 🤦🏻 will do.